### PR TITLE
added more docs to readme

### PR DESCRIPTION
**Pull request by gabbyprecious**

### CODE REVIEW

**Review by vincenzopalazzo — COMMENTED** [review]
I would like to follow the following the rust repository https://github.com/laanwj/rust-clightning-rpc for the Readme

**Comment by gabbyprecious** [comment]
Alright will work on that. with the hacking.md guideline?

File: README.md [hunk 1]
```diff
@@ -1,10 +1,10 @@
 <div align="center">
-  <h1>cln4go</h1>
+  <h1>Cln4go</h1>
```
Comment (vincenzopalazzo) [inline README.md:2]:
```suggestion
  <h1>cln4go</h1>
```

File: README.md [hunk 2]
```diff
@@ -17,25 +17,46 @@
...
 ## Packages
 
-TODO
+These are the complete list of craters supported right now
 
-## How to use
+| Crate     | Description |
+|:----------|:-----------:|
+| clng4go-client |    Package that provides means to make RPC bindings from Go code to the core lightning daemon     | 
+| cln4go-plugin |    Package that provides a plugin API to give the possibility to implement a plugin in Go     |
```
Comment (vincenzopalazzo) [inline README.md:30]:
```suggestion
| cln4go-plugin |    Package that provides a plugin API to give the possibility to implement a plugin in Go     | 
| cln4go-common |    Package that provides common interface for the monorepo. Go     | 
```

File: docs/MAINTAINERS.md [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+
+- **feat**: A new feature
+- **fix**: A bug fix
+- **deprecate**: Deprecate a feature and start to the removing process (3 official release or 1 major release)
+- **remove**: End of life for the feature.
+
+### Scopes
+
+- **rpc**: Changes related to the core lightning RPC wrapper library
+- **plugin**: Changes related to the plugin library
```
Comment (vincenzopalazzo) [inline docs/MAINTAINERS.md:95]:
```suggestion
- **plugin**: Changes related to the plugin module
- **common**: Changes related to the common module.
```

File: docs/MAINTAINERS.md [hunk 4]
```diff
@@ -0,0 +1,155 @@
...
+In particular, "core lightning Go Framework" has the possibility to release one single package at a time, or
+all together.
+
+To prepare for the release, these steps must be followed:
+
+- Bump the version number in the package before the release, and the version inside the `changelog.json` in the package root;
+    - Use a Personal Access Token or generate a [new one](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token).
+- Generate the changelog related to the package:
+    - `export GITHUB_TOKEN="your_token"`
+    - `make {changelog_rpc|changelog_plugin|changelog_plugin_macros|changelog_common|changelog}`, where`make {changelog_client|changelog_flutter|changelog}`, where
```
Comment (vincenzopalazzo) [inline docs/MAINTAINERS.md:143]:
```suggestion
    - `make {changelog_rpc|changelog_plugin|changelog_common|changelog}`, where`:
```

File: docs/MAINTAINERS.md [hunk 5]
```diff
@@ -0,0 +1,155 @@
...
+        - `changelog_client`: generate the changelog for rpc library;
+        - `changelog_plugin`: generate the changelog for plugin library;
```
Comment (vincenzopalazzo) [inline docs/MAINTAINERS.md:145]:
```suggestion
        - `changelog_plugin`: generate the changelog for plugin module;
        - `changelog_plugin_macros`: generate the changelog for common module.
```

**Review by vincenzopalazzo — CHANGES_REQUESTED** [review]
Thanks! looks Very good! 

I found some errors related to the package naming, and some typos maybe you can also open a PR when you have time to fix the typo in the repo where these examples was.

Thanks!

File: docs/MAINTAINERS.md [hunk 6]
```diff
@@ -140,9 +141,10 @@ To prepare for the release, these steps must be followed:
     - Use a Personal Access Token or generate a [new one](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token).
 - Generate the changelog related to the package:
     - `export GITHUB_TOKEN="your_token"`
-    - `make {changelog_rpc|changelog_plugin|changelog_plugin_macros|changelog_common|changelog}`, where`make {changelog_client|changelog_flutter|changelog}`, where
+    - - `make {changelog_rpc|changelog_plugin|changelog_common|changelog}`, where`:changelog_common|changelog}`, where`make {changelog_client|changelog_flutter|changelog}`, where
```
Comment (vincenzopalazzo) [inline docs/MAINTAINERS.md:144]:
```suggestion
    - `make {changelog_rpc|changelog_plugin|changelog_common|changelog}`, where:
```

**Review by vincenzopalazzo — CHANGES_REQUESTED** [review]
Just another format change

**Review by vincenzopalazzo — APPROVED** [review]

**Pull request merged**